lte/gateway/deploy/roles/magma_test_registry/tasks/main.yml

```diff
@@ -23,7 +23,6 @@
   vars:
```

Comment:
This role was introduced in #12847 as a workaround in order to distinguish between the ubuntu and debian VMs -> imho it can be removed and pkgrepo can be used again.

Reply:
Tested. Switched back to `pkgrepo` and removed `magma_test_registry`.

Reply:
noice, thank you for cleaning this up!